upgrade/v2.5/README.md

```diff
@@ -75,6 +75,10 @@ Make sure you have all the `IPPools`, `BGPPeers`, `GlobalFelixConfig`, and `Glob
 
 ## 4. Upgrade calico
 
+  > Note: If you are updating a [Canal](https://github.com/projectcalico/canal/tree/master/k8s-install)
+  deployment do not use the RBAC manifest in the following step, instead use the
+  appropriate Canal RBAC manifest.
+
   4.1. If you have RBAC enabled, apply the updated RBAC manifest `kubectl apply -f https://docs.projectcalico.org/v2.5/getting-started/kubernetes/installation/hosted/rbac-kdd.yaml`. (This will revoke access to TPRs from calico-node.)
   
   4.2. Now you can upgrade the Calico version to `v2.5.0` in your Kubernetes Calico DaemonSet. (Make sure you reboot your calico-node pods one at a time if calico-node `updateStrategy` is not set to `RollingUpdate`.) 
```

Comment:
while you're at it, can you also change this line to `v2.5.1`?

Reply:
Should I change it to a specific version or something more generic like `Now you can upgrade the Calico version in your Kubernetes Calico DaemonSet to the desired version`?

Reply:
This step could also give the option of installing the new manifest instead of just saying to change the version in your DaemonSet.  I just realized this should also say to update the install-cni image too.